### ISSUE TITLE
Missing Party Presence Listener

### ISSUE THREAD

**Issue by Amonc** [heroiclabs/nakama-dart#114]
The following snippet is from the [official doc](https://heroiclabs.com/docs/nakama/concepts/parties/#join-party). 

```
socket.onPartyPresence.listen((presence) {
  print('Joined party: ${presence.partyId}');
});
```

But unfortunately!! The function doesn't exist in the dart client.

**Comment by ilmalte** [comment]
Confirmed @Amonc, the snippet doesn't work in the current v1.1.0 available on pub.dev.

However, the implementation for that functionality has been merged in main in the PR #103. (https://github.com/heroiclabs/nakama-dart/pull/103/commits/b48276bff0f633d6a4b3f4899aa05d23e45ee40b)

I guess it will be available in the next release together with all the various bug fixes.

**Comment by ilmalte** [comment]
Ciao @Amonc, I'll close this issue as version v1.2.0 has been published, which should resolve it. 
If the problem persists, please feel free to reopen the issue with additional context. Thank you!
